### PR TITLE
Update README to reflect recent changes and additions

### PR DESCRIPTION
**Pull request by sprzedwojski**

### CODE REVIEW

File: oozie-to-airflow/README.md [hunk 1]
```diff
@@ -163,20 +163,43 @@ $ ./init.sh
...
 
-You can also specify the job.properties file, user, and output file.
+Example:
+`python o2a.py -i examples/demo -o output/demo`
 
-`python o2a.py -i <INPUT_FILE> -p <PROP_FILE> -u <USER> -o
-<OUTPUT_FILE>`
+This is the full usage guide, available by running `python o2a.py -h`
+
+```bash
```
Comment (mik-laj) [inline oozie-to-airflow/README.md]: It's not a bash script. It's only a text about a bash script.

File: oozie-to-airflow/README.md [hunk 2]
```diff
@@ -272,33 +294,25 @@ edit connections so you must add one like:
 
 `airflow connections --add --conn_id <SSH_CONN_ID> --conn_type SSH --conn_password <PASSWORD>`
 
-More information can be found on [Airflow's Website](https://airflow.apache.org/cli.html#connections)
+More information can be found on [Airflow's Website](https://airflow.apache.org/cli.html#connections).
```
Comment (mik-laj) [inline oozie-to-airflow/README.md]:
```suggestion
More information can be found on [Airflow's Documentation](https://airflow.apache.org/cli.html#connections).
```
The Airflow website does not exist yet. It's WIP.

File: oozie-to-airflow/README.md [hunk 3]
```diff
@@ -321,28 +335,117 @@ From the [Oozie documentation](https://oozie.apache.org/docs/5.1.0/WorkflowFunct
...
+**3. Streaming and pipes**
 
 Streaming and pipes are currently not supported.
 
-### Spark Example
 
-The spark example, like the other examples can be run as:
+### Pig Example
+
+The Pig example can be run as:
```
Comment (mik-laj) [inline oozie-to-airflow/README.md]:
```suggestion
The Pig example can be converted as:
```

Comment (sprzedwojski) [reply oozie-to-airflow/README.md]: I think "running" in the context of O2A means running the conversion.
What you are referring to is "running a system test", which is different in my opinion.

Besides this phrase was used before in the docs, so the original author must have thought it was appropriate.
I'd suggest leaving it as is.

File: oozie-to-airflow/README.md (hunk 2 above)
Comment (sprzedwojski) [reply oozie-to-airflow/README.md]: This was probably the previous author's "shortcut" for referring to the documentation. But I changed it to be absolutely clear.

File: oozie-to-airflow/README.md (hunk 1 above)
Comment (sprzedwojski) [reply oozie-to-airflow/README.md]: Removed

File: oozie-to-airflow/README.md (hunk 3 above)
Comment (potiuk) [reply oozie-to-airflow/README.md]: Running is fine.

**Comment by sprzedwojski** [comment]
Are there any more comments? If not, could you accept the PR? Thanks

**Pull request merged**